PKGBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: jakeroggenbuck <jakeroggenbuck2@gmail.com
+# Maintainer: cameron <cameron.bannasch@gmail>
+# Maintainer: shuzhengz <treez.zhang@gmail.com>
+#
+# This PKGBUILD was generated by `cargo aur`: https://crates.io/crates/cargo-aur
+
+pkgname=autoclockspeed-bin
+pkgver=0.1.9
+pkgrel=1
+pkgdesc="A utility to check stats about your CPU, and auto regulate clock speeds to help with either performance or battery life."
+url="https://github.com/JakeRoggenbuck/auto-clock-speed"
+license=("MIT")
+arch=("x86_64")
```

Comment:
Are we not going to support ARM then?

Reply:
Hmm, should we? I'd be open to it

Reply:
We could, but I don't have any device to test it with so yea theres that

Reply:
would a raspberry pi work?

Reply:
i think so, but we prob want more variety

Reply:
yea